nemo_skills/pipeline/train.py

```diff
@@ -254,9 +256,10 @@ def train(
             num_tasks=1,
             num_gpus=num_gpus,
             run_after=run_after,
+            task_dependencies=[prev_job_id],
```

Comment:
```suggestion
            task_dependencies=[prev_job_id] if prev_job_id is not None else None,
```
We sometimes run this script with 0 training jobs to just re-average checkpoints, so this might actually be the first job